src/app/forms/components/defect-select/defect-select.component.html

```diff
@@ -37,7 +37,9 @@ <h1 class="govuk-heading-l">Select a defect</h1>
           {{ deficiency.deficiencySubId ? '(' + deficiency.deficiencySubId + ')' : '' }}</a
         >
         <span>
-          <app-tag class="browse__category" [type]="categoryColor(deficiency.deficiencyCategory)">{{ deficiency.deficiencyCategory }}</app-tag>
+          <app-tag class="browse__category" [type]="categoryColor(deficiency.deficiencyCategory)"
+            >{{ deficiency.deficiencyCategory }}{{ deficiency.stdForProhibition ? '*' : '' }}</app-tag
```

Comment:
Don't know how much of a difference it makes, looks a bit cleaner this way
```suggestion
            >{{ deficiency.deficiencyCategory + deficiency.stdForProhibition ? '*' : '' }}</app-tag
```